src/Model/Resolver/Aggregations.php

```diff
@@ -13,13 +13,41 @@
 namespace ScandiPWA\CatalogGraphQl\Model\Resolver;
 
 use Magento\CatalogGraphQl\Model\Resolver\Aggregations as AggregationsBase;
+use Magento\CatalogGraphQl\Model\Resolver\Layer\DataProvider\Filters;
+use Magento\CatalogGraphQl\DataProvider\Product\LayeredNavigation\LayerBuilder;
 use Magento\Framework\GraphQl\Config\Element\Field;
 use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
 use Magento\Store\Api\Data\StoreInterface;
+use Magento\Catalog\Model\ResourceModel\Eav\Attribute;
 
 class Aggregations extends AggregationsBase {
+
+    /**
+     * @var Attribute
+     */
+    private Attribute $attribute;
+
+    public function __construct(
+        Filters $filtersDataProvider,
+        LayerBuilder $layerBuilder,
+        Attribute $attribute
+    )
+    {
+        parent::__construct(
+            $filtersDataProvider,
+            $layerBuilder
+        );
+
+        $this->attribute = $attribute;
+    }
+
     const PRICE_ATTR_CODE = 'price';
 
+    protected array $booleanLabels = [
+        0 => 'No',
+        1 => 'Yes'
```

Comment:
Shouldn't these labels be translatable?

Reply:
@zans-laksa please, suggest, is it fine if I check if `YES/NO` returned from the server and translate them using translations files on FE? Or there is some way to handle this on BE?
```
const translatedLabel = [BOOLEAN_YES, BOOLEAN_NO].includes(label) ? __(label) : label;
```
        
Initially the server returned 1 and 0 as boolean attributes, while data on attribute type is missing => it is insecure to rely just on 1 and 0, as they can be NOT boolean. This is why I changed `1/0` to `Yes/No` on BE after checking that the attribute is really boolean.

Reply:
But why not `true` / `false` then?
You can use the translate function on the backend too.
```
__('Yes')
```

Reply:
Info on translations on BE was helpful, thanks.

Reply:
But i'm not sure if you should harcode this.
Since without the context we cannot tell whether it's yes/no, exists/not exists, on/off, available/not available.
Even in the task it says, that it should display enable/disable.
Some labels can have different text on storefront, defined on the admin panel.
Please contact Liana and ask whether labels should be hardcoded.
Also, in the task, Yegor have a good point in his comment.
And also, if attribute values are boolean, why not to convert 1 and 0 to true and false and handle this situation on the frontend, where we can select which label to display from the context? If attribute value labels are not defined on the admin panel. Then we should use them.

Reply:
https://github.com/scandipwa/scandipwa/pull/2632
I reworked the solution and added field `is_boolean` for attributes. Now you can easily customize texts/translations just by changing this method:

```
_getBooleanValue = (label) => (+label ? __('Yes') : __('No'));
```

I agree that slider or checkbox near title will be more suitable for this case, but it is time costly and requires significant changes in logic. I asked Liana, she proposes to leave just Yes for now (which is already much better than 1). And if new requirements + design come up for boolean label display, only FE changes will be required.